mne/viz/cfc.py

```diff
@@ -0,0 +1,100 @@
+"""Functions to visualize phase/amplitude relationships in signals"""
+
+import numpy as np
+from matplotlib import pyplot as plt
```

Comment:
You should use nested imports with matplotlib. That's where the warnings come from. I also believe that is the reason the builds are timing out, since the script cannot set the correct backend for tests.